Makefile

```diff
@@ -257,13 +261,13 @@ undeploy: ## Undeploy controller from the K8s cluster specified in ~/.kube/confi
 ## Some addition to bundle creation in the bundle
 DEFAULT_ICON_BASE64 := $(shell base64 --wrap=0 ./config/assets/medik8s_blue_icon.png)
 export ICON_BASE64 ?= ${DEFAULT_ICON_BASE64}
-export BUNDLE_CSV ?="./bundle/manifests/$(OPERATOR_NAME).clusterserviceversion.yaml"
+export CSV ?="./bundle/manifests/$(OPERATOR_NAME).clusterserviceversion.yaml"
```

Comment:
Any reason to modify this var?  BUNDLE_CSV seems more appropriate as there are two CSVs in the repo. One is under _config_ and one is under _bundle_...

Reply:
I noticed that most of the other operators use `CSV`, and it is used in targets handling the bundle, so I felt OK to call it `CSV` (but mostly to simplify comparison against the other Makefiles for the same features).

Reply:
SGTM